### PR TITLE
Release 1.7.1

### PR DESCRIPTION
**Pull request by orhtej2**
## Problem

- #120 

## Solution

- Revert revert on manifest.

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by orhtej2** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -62,7 +62,7 @@ fi
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 
-if [ "$upgrade_type" == "UPGRADE_APP" ]
+if [ "$upgrade_type" == "UPGRADE_APP" ] || ynh_compare_current_package_version --comparison eq --version "1.7.1~ynh2"
```
[the remark this answers is not in the document]
Comment (orhtej2) [reply scripts/upgrade:65]: Thing is I released `1.7.1~ynh2` that's in fact 1.6.5, now updating to `1.7.1~ynh3` says it's no-upstream upgrade and refuses to pull new sources. This `||` is meant to upgrade if it's a new version OR you're stuck with borked release, n'est-ce-pas?

[the remark this answers is not in the document]
Comment (alexAubin) [reply scripts/upgrade:65]: oh okay, sounds legit :thinking: But could deserve a comment to explain why that line exists :sweat_smile:

[the remark this answers is not in the document]
Comment (orhtej2) [reply scripts/upgrade:67]: Done!

**Pull request merged**